src/utils/isDayVisible.js

```diff
@@ -0,0 +1,7 @@
+// TODO(maja): get this working for enableOutsideDays
+export default function isDayVisible(day, month, numberOfMonths) {
+  const firstDayOfFirstMonth = month.clone().startOf('month');
+  const lastDayOfLastMonth = month.clone().add(numberOfMonths - 1, 'months').endOf('month');
```

Comment:
You can avoid this work some of the time by doing an early return and putting this after it.